src/components/deployments-table/columns.tsx

```diff
@@ -81,7 +81,7 @@ export const columns: ColumnDef<Payment>[] = [
       const deployment = row.id
 
       return (
-        <Link href={`/deployments/id#${deployment}`}>
+        <Link href={`/deployments/address#${deployment}`}>
```

Comment:
Here too, on a future PR, let's reduce this to "/address#" or even just "#"...

The shorter the URLs, the better. It makes it easier for people to share them.

Reply:
Sure, I have made the change. We would need to have something before `#` for these dynamic routes. Now, the url would be `/address#`.